### PR TITLE
Ftr: add logger sample & readme

### PR DESCRIPTION
**Pull request by cjphaha**
* 添加 logger 打印到控制台， 设置日志隔离级别及输出到文件样例
* 补充readme
* 添加 .run.xml

### CODE REVIEW

**Review by Mulavar — APPROVED** [review]

**Comment by LaurenceLiZhixin** [comment]
我说怎么你的CI可以通过，manual dog head]

https://github.com/cjphaha/dubbo-go-samples/blob/config-enhance-logger/start_integrate_test.sh#L93

File: logger/README.md [hunk 1]
```diff
@@ -2,9 +2,58 @@
 
 The samples demonstrate how to configure dubbo-go logger using lumberjack
 
-### Configuration
+### Ccontents
```
Comment (xavier-niu) [inline logger/README.md:5]: double c?

File: logger/default/conf/dubbogo.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      address: 127.0.0.1:2181
+  protocols:
+    triple:
+      name: tri
+      port: 20000
+  provider:
+    registry:
+      - demoZK
+    services:
+      "greeterImpl":
```
Comment (xavier-niu) [inline logger/default/conf/dubbogo.yml:15]: The quotation should be removed.

File: logger/level/conf/dubbogo.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      address: 127.0.0.1:2181
+  protocols:
+    triple:
+      name: tri
+      port: 20000
+  provider:
+    registry:
+      - demoZK
+    services:
+      "greeterImpl":
```
Comment (xavier-niu) [inline logger/level/conf/dubbogo.yml:15]: same above

Comment (cjphaha) [reply logger/level/conf/dubbogo.yml]: thx！ i have fixed it

**Review by xavier-niu — APPROVED** [review]
LGTM

**Review by LaurenceLiZhixin — APPROVED** [review]

**Pull request merged**